ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -97,75 +97,79 @@
     {{/if}}
 
     {{#if @model}}
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-            <row.headerCell>{{t 'resources.project.title'}}</row.headerCell>
-            <row.headerCell />
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model as |model|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if (can 'read model' model)}}
+
+      <Hds::Table
+        @model={{@model}}
+        @columns={{array
+          (hash label=(t 'form.name.label'))
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'form.type.label'))
+          (hash label=(t 'resources.project.title'))
+          (hash label='')
+        }}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              <span class='hds-font-weight-semibold'>{{#if
+                  (can 'read model' B.data)
+                }}
```

Comment:
Is this format correct? The `if` statement starting on line 115 makes it a little hard to read and follow. Could we move that down and have the whole start of the `if` statement on the same line?

```suggestion
              <span class='hds-font-weight-semibold'>
                {{#if (can 'read model' B.data)}}
```

Reply:
I have fixed it.